SpeechToText: Add Overview (#8126)
DevExpress/devextreme-documentation: concepts/05 UI Components/SpeechToText/00 Overview/10 Key Features.md

- **Text Editor Integration**    

> You can integrate the SpeechToText component with any text input, including other [DevExtreme Components](/Documentation/Guide/{Framework}_Components/DevExtreme_{Framework}_Components/).

Reviewer: ```suggestion
You can integrate SpeechToText into any text input component, including other [DevExtreme Components](/Documentation/Guide/{Framework}_Components/DevExtreme_{Framework}_Components/).
```

Author: You do not integrate SpeechToText into another component, but rather with another component:
```suggestion
You can use SpeechToText together with any text input, including other [DevExtreme Components](/Documentation/Guide/{Framework}_Components/DevExtreme_{Framework}_Components/).
```
Moreover, "input" here means not just our components, but also other inputs, such as an HTML input tag

Reviewer: You can use SpeechToText with any text input, including other [DevExtreme Components](/Documentation/Guide/{Framework}_Components/DevExtreme_{Framework}_Components/).

Author: ```suggestion
You can use SpeechToText with any text input, including other [DevExtreme Components](/Documentation/Guide/{Framework}_Components/DevExtreme_{Framework}_Components/).
```

- **Language and Localization Support**    

> SpeechToText implements the Web Speech API SpeechRecognition interface, which supports transcription in multiple languages, including English, Spanish, Hindi, and Portuguese. Supported languages may vary depending on the browser.

Reviewer 2: provide a link to the info about Web Speech API SpeechRecognition interface
e.g. https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition

Reviewer 2: lets add Portuguese and other / or etc.

- **Configurable Transcription**    
You can configure [continuous transcription](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/#continuous) and [interim transcription results](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/#interimResults) to customize SpeechToText transcription behaviour. You can also bias SpeechToText results to a specified [array of words and phrases](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/#grammars).

> - **Text and Icon Customization**    

Reviewer 2: We can name this section Appearance Customization
and also mention here different modes: only icon/text and [different styling modes](link)

DevExtreme SpeechToText allows you to customize the component [icon](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#startIcon) and [text](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#startText) in both the initial and "listening" state.

> - **Custom Speech Recognizer Support**    

Reviewer 2: Let's place this section right after the 'Language and Localization Support' because they are related

> You can integrate a [custom speech recognizer](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/customSpeechRecognizer/) and use SpeechToText as a UI-only component.

Reviewer 2: We should say a few words about default speech recognition method because it's important info
